.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "env": {
```

Comment:
May I suggest the eslint [`no-var`](http://eslint.org/docs/rules/no-var.html) clause?